app/assets/javascripts/components/client/RegisterClient.jsx

```diff
@@ -0,0 +1,183 @@
+class RegisterClient extends React.Component {
+
+  constructor(props) {
+    super();
+    this.state = {success: null};
+    this.select = this.select.bind(this);
+    this.renderSelections = this.renderSelections.bind(this);
+    this.InputGroup = this.InputGroup.bind(this);
+    this.InputField = this.InputField.bind(this);
+  }
+
+  InputField({ id, label, help, ...props }) {
+    const { Checkbox, Radio, FormGroup, ControlLabel, FormControl, Button, HelpBlock } = ReactBootstrap;
+    return (
+      <FormGroup controlId={id} onChange={this.select}>
+        <ControlLabel>{label}</ControlLabel>
+        <FormControl {...props} />
+        {help && <HelpBlock>{help}</HelpBlock>}
+      </FormGroup>
+    );
+  }
+
+  renderSelections(selections) {
+    const { Checkbox, Radio, FormGroup, ControlLabel, FormControl, Button, HelpBlock } = ReactBootstrap;
+    return selections.map((selection) => {
+      return (
+        <Radio name={selection.name} value={selection.value} inline>
+          {selection.choice}
+        </Radio>
+      );
+    });
+  }
+
+  InputGroup({ id, label, help, inputs, ...props }) {
+    const { Checkbox, Radio, FormGroup, ControlLabel, FormControl, Button, HelpBlock } = ReactBootstrap;
+    return (
+      <FormGroup controlId={id} onChange={this.select}>
+        <ControlLabel >{label}</ControlLabel>
+        {this.renderSelections(inputs)}
+        {help && <HelpBlock>{help}</HelpBlock>}
+      </FormGroup>
+    );
+  }
+
+  createClient = (evt) => {
+    evt.preventDefault()
+    const payload = {
+      first_name: this.state.firstName,
```

Comment:
how is dis being sent to the API? I feel like you need to wrap it in a field like: 

```javascript
const payload = {
  client: {
    first_name: ''
    ...
  } 
}
```